thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml

```diff
@@ -0,0 +1,67 @@
+# todo: uncomment parameters after header middleware similar to 
+# `sdk-scheme-adapter` is implemented in `thirdparty-scheme-adapter`
+#parameters:
+# - $ref: ../components/parameters/ID.yaml
+# - $ref: ../components/parameters/Content-Type.yaml
+# - $ref: ../components/parameters/Date.yaml
+# - $ref: ../components/parameters/X-Forwarded-For.yaml
+# - $ref: ../components/parameters/FSPIOP-Source.yaml
+# - $ref: ../components/parameters/FSPIOP-Destination.yaml
+# - $ref: ../components/parameters/FSPIOP-Encryption.yaml
+# - $ref: ../components/parameters/FSPIOP-Signature.yaml
+# - $ref: ../components/parameters/FSPIOP-URI.yaml
+# - $ref: ../components/parameters/FSPIOP-HTTP-Method.yaml
+post:
+  description:  >
+    The HTTP request `POST /thirdpartyRequests/authorizations` is used to request the Payer to enter the
+    applicable credentials in the PISP system.
+  operationId: AuthorizationsPostRequest
```

Comment:
We're going with {HTTPVerb} {Resource} By {Param} {Param} {Param}...etc naming scheme for `operationId` for now as our rough in progress standard.

i.e PostThirdpartyRequestsAuthorizations

Reply:
Here and throughout

Reply:
Nice. Thanks @kleyow